include/FreeRTOSIPConfigDefaults.h

```diff
@@ -343,6 +343,11 @@
     #define ipconfigTCP_TIME_TO_LIVE    128
 #endif
 
+#ifndef ipconfigICMP_TIME_TO_LIVE
+    /* Set the default value suggested in RFC 1700. */
+    #define ipconfigICMP_TIME_TO_LIVE    64
```

Comment:
Yes indeed, when replying to an ICMP ping request, the stack would leave the TTL field `ucTimeToLive` untouched.
Thanks for this change.